README.md

```
@@ -1 +1,102 @@
+
+The worker can also be run as an HTTP server, providing API endpoints for proof generation and verification. To run the server:
+
+```bash
+cargo run -- --config ./templates/config.yaml
+```
+
+The server exposes the following API endpoints:
+
+- **`/api/v1/verify`**: Verifies a zk proof.
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)